raqm.c

```diff
@@ -573,7 +581,6 @@ raqm_shape_u32 (unsigned int* u32_str,
         }
         else
         {
-            int j;
```

Comment:
> Why was this declaration moved up, it is better to limit the scope of the variables.

Probably because of tiny wan about declaration in the middle of function code.